Mathlib/Data/Vector/Defs.lean

```diff
@@ -83,14 +138,32 @@ theorem cons_head_tail : ∀ v : Vector α (succ n), cons (head v) (tail v) = v
 def toList (v : Vector α n) : List α :=
   v.1
 
+@[simp] theorem mem_toVector (a : α) : ∀ (v : Vector α n), a ∈ v.toVector ↔ a ∈ v.toList :=
+  fun ⟨_, _⟩ => Vector.mem_mk.trans Array.mem_toArray
+
+@[simp] theorem mem_toList_ofVector (a : α) : ∀ (v : _root_.Vector α n),
+    a ∈ (ofVector v).toList ↔ a ∈ v :=
```

Comment:
Do you have the theorem that says `(ofVector v).toList = v.toArray.toList`?

Reply:
Ah, I had `ofVector_coe`, (which is equivalent), but not that.

Reply:
Added.

Reply:
As mentioned above, I think `ofVector_coe` should either be deleted, or `simps` should be configured to generate the right lemma.